ios-competency-test/ios-competency-test/FriendCommand.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FriendCommand.swift
+//  ios-competency-test
+//
+//  Created by Sam Gubler on 3/27/15.
+//  Copyright (c) 2015 Sam Gubler. All rights reserved.
+//
+
+import UIKit
+import Alamofire
```

Comment:
Nice! I wrote an article about how to use Alamofires JSON parsing of collections and nested objects if you are interested in reading. https://gist.github.com/jpotts18/e39ee74de84ae094b270